test/e2e/batch/batch_test.go

```diff
@@ -65,6 +67,11 @@ func testBatchInlineConfig(infinispan *v1.Infinispan) {
 	batchScript := batchString()
 	batch := createBatch(name, name, &batchScript, nil)
 
+	testKube.WaitForPods(1, tutils.SinglePodTimeout, &client.ListOptions{
+		Namespace:     tutils.Namespace,
+		LabelSelector: labels.SelectorFromSet(map[string]string{"app": "infinispan-batch-pod"}),
```

Comment:
@durgeshanaokar if two tests will work in parallel it could be handles Pods from each others
You can make `batchLabels` functions public and utilize it here